OpenRA.Game/Actor.cs

```diff
@@ -321,12 +321,12 @@ public void InflictDamage(Actor attacker, Damage damage)
 			health.InflictDamage(this, attacker, damage, false);
 		}
 
-		public void Kill(Actor attacker)
+		public void Kill(Actor attacker, HashSet<string> damageTypes = null)
```

Comment:
I'd set this to an empty hashset tbh.

Reply:
I tryed that but it didn't work, i don't remember the exact error and not at home right now, so can't redo it to see.

Reply:
Default values must be "compile time constants". It wouldn't surprise me if an empty hashset wasn't one.

Reply:
Yes, that was the error.

Reply:
Nevermind me then.